check/README.md

```diff
@@ -0,0 +1,99 @@
+# Tool for work with string tables
+
+## Glossary
+
+- `string table` a file with translation
+- `reference table` en.txt string table.
+  English is the main game language, so any changes related to text should be in that file.
+  Also this table us used as fallback for other translation.
+  This file is mostly maintained by people who change game code,
+  so the list of entries in this file should be the most actual.
+- `entry` one piece of translation, it consists of
+  - `key` a sting that represents entry in the codebase (C++, Python).
+    This string must be unique.
+  - `value` actual translation that will be shown to the user.
+
+
+### Key
+Only English chars in uppercase and dash are allowed.
+Rename of a key should be done with caution. Usually key used as is in code,
+but sometimes it could be used as prefix and code logic based on name could be present.
+
+### Value syntax
+#### Simple (single line)
+A single line of text, should not be started and ended by any whitespace character.
+
+```
+KEY
+value
+
+>>>
+value
+<<<
+```
+
+#### Full (single or multiline)
+Start and end with triple single quotes.
+You can put anything except triple singe quotes inside.
+
+```
+KEY_WITH_LEADING_SPACE
+''' value'''
+
+>>>
+ value
+<<<
+
+KEY_WITH_MULTILINE
+'''Hello
+world'''
+
+>>>
+Hello
+world
+<<<
+
+```
+
+
+#### Tags
+You can customize text with special markup tags. Not full list of tags:
+- `reference` special syntax that allows to put value from another entry.
+  You add a key in double square braces and user will see the value of that key.
+```
+KEY_WORLD
+world
+
+KEY_HELLO
+Hello [[KEY_WORLD]]!
+
+>>>
+Hello world
+<<<
+```
+
+This feature should be used to keep the translation in a consistent state.
+You define term once and use it everywhere.
+For most common used things like tech, building, etc we have more advanced type of reference: link
+
+
+- `link` special syntax that will be rendered as a link to an object wiki page.
+Supported types
+  - encyclopedia
+  - buildingtype
+  - fieldtype
+  - metertype
+  - predefinedshipdesign
+  - shiphull
+  - shippart
+  - special
+  - species
+  - tech
+  - policy
+  - value
```

Comment:
`[[value NAMED_VALUEREF_LABEL]]` doesn't make a link. Rather, it inserts the value from the script into the text.

eg. `[[value FLD_NANITE_SWARM_STRUCTURE_FLAT]]` will look up the value from the named valueref scripted as `NamedReal name = "FLD_NANITE_SWARM_STRUCTURE_FLAT" value = 5 * [[SHIP_STRUCTURE_FACTOR]])` and be replaced by whatever `5 * [[SHIP_STRUCTURE_FACTOR]]` equals, which might be just a constant in some cases, but in this case will depend on the value of a game rule.